gradle-plugins/build.gradle.kts

```diff
@@ -21,6 +21,10 @@ configurations.named("compileOnly") {
   extendsFrom(bbGradlePlugin)
 }
 
+java {
+  toolchain.languageVersion.set(JavaLanguageVersion.of(11))
+}
+
```

Comment:
Should we also have this?

https://github.com/open-telemetry/opentelemetry-java/blob/main/extensions/kotlin/build.gradle.kts#L36

Reply:
hm, after re-testing, it doesn't seem like my change gets rid of this warning.

adding the setting for kotlin doesn't seem to help either:

```
tasks.withType<KotlinCompile>().configureEach {
  kotlinOptions {
    jvmTarget = "11"
  }
}
```

googling around I found https://github.com/gradle/gradle/issues/18935 which seems to be the issue here